README.md

```diff
@@ -1,3 +1,6 @@
 #Data Structures and Algorithms, the Java Version 
 
-1. [ArrayShift](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/src/main/java/ArrayShift.java)
\ No newline at end of file
+1. [ArrayShift](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/src/main/java/ArrayShift.java)
+    ![whiteboard](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/assets/IMG_1614.jpeg)
```

Comment:
These file paths don't work! Try using ./assets/IMG_1614.jpeg instead.

Reply:
Thank you! I think I just copied and pasted in IDE, but Im implementing this and we will see how it goes!